tests/agents/test_max_height.py

```diff
@@ -0,0 +1,50 @@
+import holodeck
+import uuid
+from copy import deepcopy
+
+from tests.utils.equality import almost_equal
+
+uav_config = {
+    "name": "test_max_height",
+    "world": "TestWorld",
+    "main_agent": "uav0",
+    "agents": [
+        {
+            "agent_name": "uav0",
+            "agent_type": "UavAgent",
+            "sensors": [
+                {
+                    "sensor_type": "LocationSensor",
+                }
+            ],
+            "control_scheme": 0,
+            "location": [0, 0, 2],
+            "max_height": 500 - 11.8283691 # in centimeters
```

Comment:
This should be in meters (see my review on the engine side to convert back to cm). Also, what's with the additional 11 cm?